lib/src/over_react_test/jacket.dart

```diff
@@ -105,6 +113,11 @@ class TestJacket<T extends react.Component> {
     return over_react.getDartComponent(_renderedInstance) as T;
   }
 
+  /// Returns if the jacket component is mounted or not.
+  bool isMounted() {
```

Comment:
Could we make this a getter instead? I want to eventually do the same with a lot of the other methods in this class...

Reply:
Surreee thing.